scylla-cql/src/frame/response/result.rs

```diff
@@ -161,8 +161,8 @@ impl CqlValue {
         self.as_cql_date().and_then(|date| date.try_into().ok())
     }
 
-    #[cfg(feature = "time")]
-    pub fn as_date(&self) -> Option<time::Date> {
+    #[cfg(feature = "time-03")]
+    pub fn as_date(&self) -> Option<time_03::Date> {
         self.as_cql_date().and_then(|date| date.try_into().ok())
     }
```

Comment:
Well, it looks like it should be removed then. Otherwise, we would need to introduce some weird version-specific names for the corresponding methods. WDYT?

Reply:
@Lorak-mmk It seems your answer is awaited.

Reply:
We obviously can't just remove the method - how would we handle dates then? I'm not sure what the best solution is here. Current one is not feasible - why put those crates behing per-version features when we can't add a new one? Any ideas?

Reply:
I don't immediately see any better solution than per-version methods.

Reply:
> We obviously can't just remove the method - how would we handle dates then?

Notice that there are no corresponding methods for `num_bigint::BigInt`. These methods are not really necessary - they are just handy utility functions. Removing them would require the users to explicitly convert `CqlValue` to `value::CqlDate` (using `as_cql_date` method), which then, could be converted to specific `time::Date` version (using the `TryInto` implementation).

Knowing that we could possibly remove the function, do you still think that introducing per-version methods is a better solution?

Reply:
In that case yes, I think we should remove the method.